ls1intum/Apollon
# Feature/floating action buttons

This pull request adds floating action buttons to Apollon to make editing and deleting elements more easily accessible.

### Checklist

### Motivation and Context
So far, editing elements was only available through rather unintuitive double clicks. Now, every element shows a set of floating action buttons when either selected or hovered.


### Steps for Testing
Select any element on the canvas or hover it via your cursor. You should now see two floating action buttons, one for editing, one for deleting said element on the canvas.

### Screenshots
<img width="277" alt="Screenshot 2023-11-26 at 20 13 35" src="https://github.com/ls1intum/Apollon/assets/143808484/2515acf8-f705-4091-864b-8c3c2cd7ec87">
